### PR TITLE
TW-916: improve size image preview link

### PR DESCRIPTION
**Pull request by imGok**
#916 
Large (h > 200)
image

Small (h <= 200)
image

### CODE REVIEW

**Comment by hoangdat** [comment]
hi @imGok github preview have the problem

image

**Comment by sherlockvn** [comment]
The color of message is impact
image

**Comment by hoangdat** [comment]
@imGok when it runs on my side

image

**Comment by sherlockvn** [comment]
> @imGok when it runs on my side, @sherlockvn please help him on it (maybe something relate to screen resolution)
> 
> image

I have tried it in local but don't got your error

https://github.com/linagora/twake-on-matrix/assets/43041967/3a863798-7cd3-450e-ae4b-81b1dbadbec1

**Comment by hoangdat** [comment]
@drminh2807 please review it

File: lib/widgets/twake_components/twake_preview_link/twake_link_preview_item.dart [hunk 1]
```diff
@@ -32,11 +30,155 @@ class TwakeLinkPreviewItem extends StatelessWidget {
...
+              ),
+            ),
+            child: Container(
+              constraints: const BoxConstraints(
+                maxHeight: TwakeLinkPreviewItemStyle.maxHeightPreviewLarge,
+              ),
+              child: Expanded(
+                child: MxcImage(
+                  uri: urlPreviewPresentation.imageUri,
+                  fit: BoxFit.fitWidth,
```
Comment (sherlockvn) [inline lib/widgets/twake_components/twake_preview_link/twake_link_preview_item.dart:128]: why don't you use cover, because maybe image become too large

**Comment by sherlockvn** [comment]
Sr but this image link is not cover well. https://www.youtube.com/watch?v=JkqLdK85vj0
image

**Comment by imGok** [comment]
@sherlockvn Please check again after clearing your browser cache! :)

**Comment by sherlockvn** [comment]
> @sherlockvn Please check again after clearing your browser cache! :)

I don't clear cache but it's work :) amazing

**Review by sherlockvn — APPROVED** [review]

**Pull request merged**